source/docs/software/labview/resources/labview-resources.rst

```diff
@@ -10,7 +10,9 @@ LabVIEW Resources
 LabVIEW Basics
 --------------
 
-NI provides a `combination of videos <https://www.ni.com/academic/students/learn-labview/>`_ and `traditional text/picture tutorials on the basics of LabVIEW <https://www.ni.com/getting-started/labview-basics/>`_. These tutorials can help you get acquainted with the LabVIEW environment and the basics of the graphical, dataflow programing model used in LabVIEW.
+NI provides `tutorials on the basics of LabVIEW <https://www.ni.com/getting-started/labview-basics/>`_. These tutorials can help you get acquainted with the LabVIEW environment and the basics of the graphical, dataflow programing model used in LabVIEW.
+
+.. todo:: Use student videos if kevin gets the link fixed
```

Comment:
```suggestion
.. todo:: Add academic videos when link fixed https://www.ni.com/academic/students/learn-labview/
```

Reply:
Linkcheck will check todo links

Reply:
even if it's not formatted as a link?

Reply:
Correct.

Reply:
Edited my suggestion and confirmed it passed linkcheckdiff